releasenotes-builder/src/test/java/com/github/checkstyle/internal/AbstractPathTestSupport.java

```diff
@@ -42,6 +42,23 @@ public abstract class AbstractPathTestSupport {
     protected static final String MISC = Constants.MISCELLANEOUS_LABEL;
     protected static final String BREAKING = Constants.BREAKING_COMPATIBILITY_LABEL;
 
+    protected static final String MSG_EXECUTION_SUCCEEDED = System.lineSeparator()
+            + "Execution succeeded!" + System.lineSeparator();
+
+    protected static final String MSG_RELEASE_IS_MINOR =
+        "\nGeneration ends with 1 errors.\n\n[ERROR] Validation of release number failed. "
```

Comment:
This is not just release is minor. It has the number of errors too, which seems a bit too specific for general usage. These 2 should be split, the number of errors should either be hard coded or find some way to make it clear the number and easy to use for others.

Reply:
I split them and created `getExecutionFailedMessage(int amountOfErrors)` method.